### DISCUSSION TITLE
Windows Command promt closes when I try to install serverpod

### DISCUSSION THREAD

**Discussion by Zingo21** [serverpod/serverpod#853] [Q&A]
**Describe the bug**

When I type the command for installing serverpod on Windows command prompt and press enter, the whole window closes. And it haven't downloaded serverpod.

**To Reproduce**
1. Type in ``` dart pub global activate serverpod_cli ```
2. Press enter

3. See the window close by itself

**Expected behavior**

I expected that it would begin to download serverpod

**Screenshots**

**Serverpod versions**
 -  _CLI Version:_ Run `serverpod version` to determine the version of installed tooling.
 - _Library version:_ Check the pubspec.yaml file for the version of Serverpod you are using.

**Platform information**
On which platforms do you experience the issue. Please provide as much information here as you can. Where are you running the server, which front end device are you using?

HP Elitedesk 800 G2
Windows 10

**Additional context**

**Comment by vlidholt** [comment]
This sounds more like an issue with Dart/Flutter. Make sure that you have the latest version of Flutter installed and try running `flutter doctor` to see if there are any issues with your install.
